sdk/core/System.ClientModel/src/Pipeline/ClientLoggingPolicy.cs

```diff
@@ -0,0 +1,360 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Internal;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace System.ClientModel.Primitives;
+
+/// <summary>
+/// A <see cref="PipelinePolicy"/> used by a <see cref="ClientPipeline"/> to
+/// log request and response information.
+/// </summary>
+public class ClientLoggingPolicy : PipelinePolicy
+{
+    private const double RequestTooLongSeconds = 3.0; // sec
```

Comment:
Could we put these defaults in the options type?

Reply:
I think it's more awkward to set defaults in the options type since we're using nullability to determine if something has been changed.

Reply:
I'm thinking less of setting defaults and more about storing the constants in a central place close to the options that might use them.  Does it still feel awkward to you even from that perspective?

Reply:
Are we expecting all policies to use the same defaults? If so, then I guess they would need to be internal on the options type